websites/A/Aniorb/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+      "name": "ICEPrey",
+      "id": "256973318588989440"
+    },
+    "service": "Aniorb",
+    "description": {
+      "en": "Aniorb is a free anime website with no ads and the highest quality animes."
+    },
+    "url": "aniorb.me",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/9qhIuod.png",
+    "thumbnail": "https://i.imgur.com/THrtHoe.png",
+    "color": "#000000",
+    "tags": ["aniorb", "ani", "orb", "4anime", "animelist", "aniwatch", "animekisa", "gogoanime", "anime" , "watching" , "popular", "streaming", "free", "fast", "1080p", "website"],

```

Comment:
why tags like 4anime animelist aniwatch animekisa etc.?

Reply:
better search?

Reply:
they need to be tags related to this service not to other services :/